docs/table/table_and_dataframes.rst

```diff
@@ -39,15 +39,17 @@ VOTable data. This means that when you read a FITS or VOTable file into a table
 write it back out, the data will be effectively identical. This is made possible by
 robust support for table and column metadata which allows storing and propagating common
 column information such as the unit, description, and format. For VOTable data, more
-esoteric information like the UCD is maintained.
+information like the UCD is maintained.
 
-FITS and VOTable are two important file formats in astronomy, but the idea of lossless
-representation of physical data is critical in many other contexts as well, for instance
-when retrieving data from the numerous data catalogs provided by `astroquery
-<https://astroquery.readthedocs.io/en/latest/>`_.
+FITS and VOTable are two important data formats in astronomy, but the idea of lossless
+representation of physical data is critical in other contexts as well. For
+example, accessing metadata like units is a key component of using the
+`astroquery <https://astroquery.readthedocs.io/en/latest/>`_ service for astronomical
+data analysis.
```

Comment:
I still don't get the astroquery reference, and why this is a key component for astroquery. We could return whatever comes back from the server, typically a VOTable or csv or whatever text file object. It's just a compatibility thing that we went a step further and parse the results into an astropy Table. 


(and returning pandas definitely brought up by a couple of modules but we pushed them to do astropy Tables instead to be more consistent with the older modules).

Reply:
IMO a key feature of astroquery is exactly the hard work of parsing heterogenous results from data providers into a uniform and predictable data structure (QTable) that is easy for users to work with. Many of those datasets include physical quantities with units, column descriptions, or other metadata, so if astroquery returned a DataFrame then that information would be lost.

That said, if you really don't want this shout-out to astroquery then I can replace it with a different example.

Reply:
Seconded - for me part of the beauty of astroquery is exactly that it produces a table with usable columns!

Reply:
Yes, I think if you can go with a different example it would be OK as I'm just not comfortable with the 
` is a key component of using the astroquery service for astronomical data analysis` summary, and nitpicking the sentence away is not very constructive.